news/2019-12-04-new-featured-artist-kurokotei.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Kurokotei"
+date: 2019-12-04 10:00:00 +0000
+---
+
+**Kurokotei** soars in from a collapsed galaxy to join the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/58/header.jpg)
+
+We're super proud to announce that **Kurokotei** is the latest name to join the ever-expanding Featured Artist roster, complete with a selection from her latest album, *Into the Void*. As an extra bonus, **Kurokotei** is also a long-time member of the osu!community, an established mapper and top-end osu!taiko player in her own right. There's something extra special about bringing people aboard from the community!
+
+The older among us might remember **Kuroktei**'s track titled *Galaxy Collapse*, made famous in osu! by [the now Loved set](https://osu.ppy.sh/beatmapsets/396221#osu/862088). It remains to this day, one of the hardest beatmaps ever made, and we all know just how much the osu! community loves its ridiculously difficult maps.
```

Comment:
shoudl mention her taiko map that came before this one, it was popular even among standard players https://osu.ppy.sh/beatmapsets/328117 and I think the hardest ranked taiko map at the time ?

also is 3 years "old" now 🤔

Reply:
good call and yes, it is, unfortunately :(